common/src/test/java/tachyon/conf/TachyonConfTest.java

```diff
@@ -64,204 +64,204 @@ public void beforeTests() {
-    Assert.assertTrue(!booleanValue);
+    Assert.assertEquals(booleanValue, false);
 
     booleanValue = sDefaultTachyonConf.getBoolean(Constants.IN_TEST_MODE);
-    Assert.assertTrue(!booleanValue);
+    Assert.assertEquals(booleanValue, false);
 
     booleanValue = sDefaultTachyonConf.getBoolean(Constants.ASYNC_ENABLED);
-    Assert.assertTrue(!booleanValue);
+    Assert.assertEquals(booleanValue, false);
 
     int intValue = sDefaultTachyonConf.getInt(Constants.MAX_COLUMNS);
-    Assert.assertTrue(intValue == 1000);
+    Assert.assertEquals(intValue, 1000);
 
     intValue = sDefaultTachyonConf.getInt(Constants.HOST_RESOLUTION_TIMEOUT_MS);
     Assert.assertEquals(Constants.DEFAULT_HOST_RESOLUTION_TIMEOUT_MS, intValue);
 
     long longBytesValue = sDefaultTachyonConf.getBytes(Constants.MAX_TABLE_METADATA_BYTE);
-    Assert.assertTrue(longBytesValue == Constants.MB * 5);
+    Assert.assertEquals(longBytesValue, Constants.MB * 5);
   }
 
   @Test
   public void testMasterDefault() {
     String tachyonHome = sDefaultTachyonConf.get(Constants.TACHYON_HOME);
-    Assert.assertTrue(tachyonHome != null);
-    Assert.assertTrue("/mnt/tachyon_default_home".equals(tachyonHome));
+    Assert.assertNotNull(tachyonHome);
+    Assert.assertEquals("/mnt/tachyon_default_home", tachyonHome);
 
     String value = sDefaultTachyonConf.get(Constants.MASTER_JOURNAL_FOLDER);
-    Assert.assertTrue(value != null);
-    Assert.assertTrue((tachyonHome + "/journal/").equals(value));
+    Assert.assertNotNull(value);
+    Assert.assertEquals(tachyonHome + "/journal/", value);
 
     value = sDefaultTachyonConf.get(Constants.MASTER_HOSTNAME);
-    Assert.assertTrue(value != null);
-    Assert.assertTrue(NetworkAddressUtils.getLocalHostName(100).equals(value));
+    Assert.assertNotNull(value);
+    Assert.assertEquals(NetworkAddressUtils.getLocalHostName(100), value);
 
     value = sDefaultTachyonConf.get(Constants.MASTER_TEMPORARY_FOLDER);
-    Assert.assertTrue(value != null);
-    Assert.assertTrue("/tmp".equals(value));
+    Assert.assertNotNull(value);
+    Assert.assertEquals("/tmp", value);
 
     value = sDefaultTachyonConf.get(Constants.MASTER_FORMAT_FILE_PREFIX);
-    Assert.assertTrue(value != null);
-    Assert.assertTrue(Constants.FORMAT_FILE_PREFIX.equals(value));
+    Assert.assertNotNull(value);
+    Assert.assertEquals(Constants.FORMAT_FILE_PREFIX, value);
 
     value = sDefaultTachyonConf.get(Constants.MASTER_ADDRESS);
-    Assert.assertTrue(value != null);
+    Assert.assertNotNull(value);
 
     value = sDefaultTachyonConf.get(Constants.MASTER_BIND_HOST);
-    Assert.assertTrue(value != null);
-    Assert.assertTrue(NetworkAddressUtils.WILDCARD_ADDRESS.equals(value));
+    Assert.assertNotNull(value);
+    Assert.assertEquals(NetworkAddressUtils.WILDCARD_ADDRESS, value);
 
     int intValue = sDefaultTachyonConf.getInt(Constants.MASTER_PORT);
-    Assert.assertTrue(intValue == 19998);
+    Assert.assertEquals(intValue, 19998);
```

Comment:
`Assert.assertEquals(intValue, 19998);` -> `Assert.assertEquals(19998, intValue);`?
we shall put the expected value as the first parameter, and actual value as the second parameter to generate correct assertion failure message.